packages/terra-form-select/src/single/Frame.jsx

```diff
@@ -265,7 +280,7 @@ class Frame extends React.Component {
           _onBlur();
         }
       }, 10);
-    // Modern browsers support event.relatedTarget
+      // Modern browsers support event.relatedTarget
```

Comment:
nit : extra-space

Reply:
I think they were generated after auto-fix, linter didn't complain though. So I think it's fine.